Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:latest
+
+RUN apt-get update && apt-get install -y \
+    npm \
+    php \
+    vim \
+    ruby \
+    time \
+    clang \
+    pypy3 \
+    golang \
+    nodejs \
```

Comment:
I think the nodejs in Ubuntu repository wasn't the official way of installation. I'm suspecting if it is the official channel 🫨